public_dropin_gpu_environments/nim_llm/dr_requirements.txt

```diff
@@ -71,8 +100,11 @@ tqdm==4.66.6
 trafaret==2.1.1
 typing-extensions==4.12.2
 tzdata==2024.2
-urllib3==2.2.3
+urllib3==1.26.20
```

Comment:
dr-storage is forcing us to downgrade this package 😢

Reply:
Remember we had issues with urllib3 1 vs 2 incompat last year. No idea if this is any kind of threat but I'd say need to verify that this all still works with urllib 1